shorten_service/app.py

```diff
@@ -36,10 +51,15 @@ def lambda_handler(event, context):
            }
 
 
-       # TODO (tarik): Replace this with base62 logic
-       short_id = uuid.uuid4().hex[:7]
-
+       response = short_url_table.update_item(
+           Key={'id': 'counter'},
```

Comment:
It looks like you made a decision to keep the counter integer in dynamodb. It's okay for this POC implementation but we should consider pros and cons of Dynamo for this. We can get a better latency with Redis maybe. It's okay to merge this right now but take a note for this.